File: dts-playlist/dts-playlist-inf-file/src/main/java/com/gugu/dts/playlist/inf/entity/MusicLibrary.java
package com.gugu.dts.playlist.inf.entity;

/**
 * Table: music_library
 */
public class MusicLibrary {
    /**
     * Column: id
     */
    private Integer id;

    /**
     * Column: name
     */
    private String name;

    /**
     * Column: path
     */
    private String path;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name == null ? null : name.trim();
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path == null ? null : path.trim();
    }
}

File: dts-playlist/dts-playlist-core/src/main/kotlin/com/gugu/dts/playlist/core/entity/MusicLibrary.kt
package com.gugu.dts.playlist.core.entity

import com.gugu.dts.playlist.api.`object`.IMusicLibrary
import com.gugu.dts.playlist.api.`object`.ISong

class MusicLibrary(
        override val name: String,
        override val path: String,
        override val songs: List<ISong>
) : IMusicLibrary 